src/dmd/iasmdmd.d

```diff
@@ -1430,8 +1430,7 @@ code *asm_emit(Loc loc,
             ADDFWAIT &&
             !(ptb.pptb0.usFlags & _nfwait))
             pc.Iflags |= CFwait;
-        else if ((ptb.pptb0.usFlags & _fwait) &&
-                 config.target_cpu >= TARGET_80386)
```

Comment:
There is no other `config.target_cpu` set, other than `TARGET_PentiumPro` in DMD. This can be moved to backend anyway.